repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapper.scala

```diff
@@ -103,29 +88,31 @@ class HttpServletRequestWrapper(originalRequest: HttpServletRequest, inputStream
     }
   }
 
+  override def addHeader(headerName: String, headerValue: String): Unit = {
+    val existingHeaders: List[String] = getHeadersScala(headerName) //this has to be done before we remove from the list,
+    // because getting this list is partially based on the contents of the removed list
+    removedHeaders = removedHeaders - headerName
+    headerMap = headerMap + (headerName -> (existingHeaders :+ headerValue))
+  }
+
```

Comment:
I have had it do that to me a few times too. Sometimes just running the formatter again will put things back in their place, but if it is moving multiple things around, then you never know what it is going to move. I was just curious if there was a reason.